updating/updating-restricted-network-cluster.adoc

```diff
@@ -38,11 +38,23 @@ include::modules/cli-installing-cli.adoc[leveloffset=+3]
 
 // this file doesn't exist, so I'm including the one that should pick up more changes from Clayton's PR - modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
 
-include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
+include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+3]
 
-include::modules/update-mirror-repository.adoc[leveloffset=+2]
+[id="update-mirror-repository_{context}"]
```

Comment:
Nice job including the ID above, but when you do IDs in assembly files, you don't add the `_{context}` piece. That piece is only used in module files.